### ISSUE TITLE
doesn't work on fresh install of windows 10

### ISSUE THREAD

**Issue by coclav** [atom/rcedit#3]
I don't know if this will really help, but I couldn't get to make rcedit work on a fresh install of windows 10 (in parallel desktop)

- first I had messages that `MSVCP100.dll is missing`, and `MSVCR100.dll is missing` as well. I found them somewhere else in the windows folder, copied them to SYSWOW64 folder; 
- the next error was a `The application was unable to start correctly (0xc000007b)`

I have installed all the c++ and whatnot windows SDK I could find... without success

I gave up at this stage because at this point most of the day had gone, and I switched to resourcehacker. It's less elegant solution but does the job.

**Comment by szwacz** [comment]
+1

**Comment by paulcbetts** [comment]
If you copied a file from somewhere in the Windows folder to SysWow64, you probably copied in a 64-bit DLL into the SysWow64 (aka 32-bit) directory, which is why you got the error. It would be cool to flip the executable to static link so you wouldn't have to track down the MSVC runtime

**Comment by feross** [comment]
Looks like there's already a PR to fix this here: https://github.com/atom/rcedit/pull/11

In case it helps future bug hunters, I'll share my full error output (on Windows 10) which was different than the original reporter's, but almost certainly caused by the same issue:

```
Failed with exit code: 4294967295
Output:
System.AggregateException: One or more errors occurred. ---> System.Exception: Failed to modify resources, command invoked was: 'C:\Users\Feross Aboukhadijeh\code\webtorrent-desktop\node_modules\electron-winstaller\vendor\rcedit.exe "C:\Users\Feross Aboukhadijeh\code\webtorrent-desktop\dist\Setup.exe" --set-version-string "CompanyName" "The WebTorrent Project" --set-version-string "LegalCopyright" "Copyright � 2016 The WebTorrent Project" --set-version-string "FileDescription" "WebTorrent" --set-version-string "ProductName" "WebTorrent" --set-file-version 0.2.0 --set-product-version 0.2.0 --set-icon "C:\Users\Feross Aboukhadijeh\code\webtorrent-desktop\static\WebTorrent.ico"'

Output was:

   at Squirrel.Update.Program.<setPEVersionInfoAndIcon>d__a2.MoveNext()
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at Squirrel.Update.Program.<>c__DisplayClass78.<Releasify>b__72()
   at Squirrel.Utility.<>c__DisplayClassc.<Retry>b__b()
   at Squirrel.Utility.Retry[T](Func`1 block, Int32 retries)
   at Squirrel.Utility.Retry(Action block, Int32 retries)
   at Squirrel.Update.Program.Releasify(String package, String targetDir, String packagesDir, String bootstrapperExe, String backgroundGif, String signingOpts, String baseUrl, String setupIcon, Boolean generateMsi)
   at Squirrel.Update.Program.executeCommandLine(String[] args)
   at Squirrel.Update.Program.main(String[] args)
   at Squirrel.Update.Program.Main(String[] args)
---> (Inner Exception #0) System.Exception: Failed to modify resources, command invoked was: 'C:\Users\Feross Aboukhadijeh\code\webtorrent-desktop\node_modules\electron-winstaller\vendor\rcedit.exe "C:\Users\Feross Aboukhadijeh\code\webtorrent-desktop\dist\Setup.exe" --set-version-string "CompanyName" "The WebTorrent Project" --set-version-string "LegalCopyright" "Copyright � 2016 The WebTorrent Project" --set-version-string "FileDescription" "WebTorrent" --set-version-string "ProductName" "WebTorrent" --set-file-version 0.2.0 --set-product-version 0.2.0 --set-icon "C:\Users\Feross Aboukhadijeh\code\webtorrent-desktop\static\WebTorrent.ico"'

Output was:

   at Squirrel.Update.Program.<setPEVersionInfoAndIcon>d__a2.MoveNext()<---
```

**Comment by zcbenz** [comment]
Closed with #11.
